.github/workflows/autoupdate-spec.yaml

```diff
@@ -0,0 +1,89 @@
+name: Update API Commands
+on:
+  schedule:
+   - cron: 30 8 * * TUE
+  workflow_dispatch:
+jobs:
+  update-sdk:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GitHubSecurityLab/actions-permissions/monitor@v1
+        with:
+          config: ${{ vars.PERMISSIONS_CONFIG }}
+      - name: Checkout
+        uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+      - run:  make update-openapi-spec
+      - name: Verify Changed files
+        uses: tj-actions/verify-changed-files@6b59fb7cbb8d9a6ecc10ee556496d0078a9ed957
+        id: verify-changed-files
+        with:
+          files: |
+             ./tools/api-generator/spec.yaml
+      - run:  make gen-api-commands
+        if: steps.verify-changed-files.outputs.files_changed == 'true'
+      - name: Find JIRA ticket
+        id: find
+        if: steps.verify-changed-files.outputs.files_changed == 'true'
+        uses: mongodb/apix-action/find-jira@d74254f04d46b8e94ef22248607d47225c8a171a
+        with:
+          token: ${{ secrets.JIRA_API_TOKEN }}
+          jql: project = CLOUDP AND status NOT IN (Closed, Resolved) AND summary ~ "Update Autogenerated Commands"
+      - name: Set JIRA ticket (find)
+        if: (steps.verify-changed-files.outputs.files_changed == 'true') && (steps.find.outputs.found == 'true')
+        run: |
+          echo "JIRA_KEY=${{steps.find.outputs.issue-key}}" >> "$GITHUB_ENV"
+      - name: Create JIRA ticket
+        uses: mongodb/apix-action/create-jira@d74254f04d46b8e94ef22248607d47225c8a171a
+        id: create
+        if: (steps.verify-changed-files.outputs.files_changed == 'true') && (steps.find.outputs.found == 'false')
+        with:
+          token: ${{ secrets.JIRA_API_TOKEN }}
+          project-key: CLOUDP
+          summary: Update Autogenerated Commands
+          issuetype: Story
+          description: Update Autogenerated Commands
```

Comment:
```suggestion
          description: [AtlasCLI] Update Autogenerated Commands
```

Reply:
I think, this only makes sense for the title, we already have atlascli as component